sources/github/index.js

```diff
@@ -20,7 +20,10 @@ module.exports.ignoreProjects = ({ html_url }) =>
 
 module.exports.ignoreBotUsers = ({ user: { login } = {} }) => !login.startsWith('dependabot')
 
-module.exports.enrichIssues = async ({ issues, start, end, team }) => {
+module.exports.enrichIssues = async ({ issues: _issues, start, end, team }) => {
+  // filter stale PRs likely being deleted after being closed for a while
+  // TODO: resort to a more reliable way to detect stale PRs
+  const issues = _issues.filter((v) => !v.closed_at || v.closed_at.split('T')[0] >= v.updated_at.split('T')[0])
```

Comment:
(I'm not sure here) can't you use `v.state` or a combination of `v.state && || v.closed_at`? haven't seen examples of real PRs to understand how the data comes tho..

Reply:
example https://github.com/eqworks/snoke/pull/908

![close/delete](https://user-images.githubusercontent.com/2837532/104968458-e58a5b80-59b3-11eb-902b-4f8508b4bb26.png)

the associated branch deletion triggered another update, which is _likely_ false alarm

re: `v.state`, by my observation, if `v.state` isn't "closed", then `v.close_at` is `null`, which means that (conceptually), `!v.closed_at` is basically `v.state != 'closed'`.